CMakeLists.txt

```
@@ -876,6 +876,7 @@ cmake_dependent_option(
 # feature by default We dont currently document this feature because we don't
 # Suspect users building from source will need this
 add_definitions(-DFLASHATTENTION_DISABLE_ALIBI)
+add_definitions(-DFLASH_NAMESPACE=pytorch_flash)
```

```suggestion
add_definitions(-DFLASHATTENTION_DISABLE_SOFTCAP)
add_definitions(-DFLASH_NAMESPACE=pytorch_flash)
```
We do not support softcaps after all.